tests/unit/generators/globalConfig_generator/test_create_minimal_globalConfig.py

```diff
@@ -0,0 +1,122 @@
+from pytest import fixture
+from unittest.mock import patch, MagicMock
+from splunk_add_on_ucc_framework.generators.globalConfig_generator import (
+    MinimalGlobalConfig,
+)
+
+
+@fixture
+def global_config():
+    return None
+
+
+@fixture
+def app_manifest():
+    mock_manifest = MagicMock()
+    mock_manifest.get_addon_name.return_value = "test_addon"
+    mock_manifest.get_addon_version.return_value = "1.0.0"
+    mock_manifest.get_title.return_value = "Test title"
+    return mock_manifest
+
+
+@fixture
+def input_dir(tmp_path):
+    return str(tmp_path / "input_dir")
+
+
+@fixture
+def output_dir(tmp_path):
+    return str(tmp_path / "output_dir")
+
+
+@fixture
+def ucc_dir(tmp_path):
+    return str(tmp_path / "ucc_dir")
```

Comment:
We have this defined in multiple places (i.e. https://github.com/splunk/addonfactory-ucc-generator/blob/728642d3c770180ff9ccc46c51c5019e5a0e08ab/tests/unit/generators/xml_files/test_create_configuration_xml.py#L18-L35), can you please create a separate `refactor` PR and move all those out of specific test files into `conftest`? I'd like that PR to be merged before this one.

Reply:
See https://github.com/splunk/addonfactory-ucc-generator/pull/1646